src/main/java/at/hannibal2/skyhanni/mixins/hooks/RendererLivingEntityHook.kt

```diff
@@ -57,11 +52,13 @@ class RendererLivingEntityHook {
 
     private fun isCoolPerson(userName: String?): Boolean {
         if (!LorenzUtils.inSkyBlock) return false
+        if (!config.flipContributors || LorenzUtils.isAprilFoolsDay) return false
```

Comment:
Shouldnt this be `&& !LorenzUtils.isAprilFoolsDay`

Reply:
It should, I'll fix it when I wake up